controllers/placementdecision/placementdecision_controller.go

```diff
@@ -124,15 +125,16 @@ func (r *PlacementDecisionReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 }
 
-func (r *PlacementDecisionReconciler) processPlacementDecisionUpdate(placement *clusterv1alpha1.Placement) (ctrl.Result, error) {
```

Comment:
This file should really be renamed to placement_controller and all of the various incorrect comments should be cleaned up. It's a bit hard to wrap your head around this file and what it is and is not for in its current state. Not suggesting for this PR. :-)